File: tests/machine/x/lua/cast_struct.lua
function __print(...)
    local args = {...}
    local parts = {}
    for i,a in ipairs(args) do
        if a ~= nil and a ~= '' then parts[#parts+1] = tostring(a) end
    end
    print(table.concat(parts, ' '))
end
Todo = {}
Todo.__index = Todo
function Todo.new(o)
  o = o or {}
  setmetatable(o, Todo)
  return o
end

todo = Todo.new({["title"]="hi"})
__print(todo.title)

File: archived/tests/transpiler/x/java/cast_struct.java
public class Main {
    static class Todo {
        String title;
        Todo(String title) {
            this.title = title;
        }
    }

    static Todo todo = new Todo("hi");
    static class Data1 {
        String title;
        Data1(String title) {
            this.title = title;
        }
    }


    public static void main(String[] args) {
        System.out.println(todo.title);
    }
}
